contest/contributions/trendecon.yml

```
@@ -0,0 +1,27 @@
+# Contribution's title, author, abstract
+title: Daily economic indicators based on Google searches # title case
+author: Angelica Becerra, Vera Z. Eichenauer, Ronald Indergand, Stefan Legge,
+  Isabel Martinez, Nina Mühlebach, Furkan Oguz, Christoph Sax,
+  Kristina Schuepbach, Severin Thöni, Uwe Thümmel
+abstract: |
+  During the Covid-19 pandemic, information and the (economic and social)
+  situation has changed rapidly. Traditional (economic) indicators are not
+  sufficiently frequent to monitor and forecast (economic and social) activity
+  at high frequency. We use Google search trends to overcome this data gap and
+  create meaningful indicators. We extract daily search data on keywords
+  reflecting consumers' perception of the economic situation. The indicators
+  are available at [www.trendecon.org](https://www.trendecon.org).
+
+  An accompaning R package contains the code to construct long daily time series
+  from Google Trends. Robustness of the data is achieved by querying Google
+  mulitple times. The queries are sampled at daily, weekly and monthy
+  frequencies and then harmonized such that the long term trend is preserved. A
+  more detailed methodological description is given on the
+  [website](https://www.trendecon.org/#method). We are currently summarizing
+  these results in a research paper.
+# Short name to be used in the website Gallery menu (can be the same as title)
+menu_entry: Daily economic indicators based on Google searches
```

Shorter `menu_entry` better fit to the Gallery menu
```suggestion
menu_entry: Daily economic indicators
```